public/app/core/logsModel.ts

```diff
@@ -784,7 +784,7 @@ export function queryLogsSample<TQuery extends DataQuery, TOptions extends DataS
 }
 
 function getIntervalInfo(scopedVars: ScopedVars, timespanMs: number): { interval: string; intervalMs?: number } {
-  if (scopedVars.__interval) {
+  if (scopedVars.__interval_ms) {
```

Comment:
is this a real change, or an artifact of changing our setup?

Reply:
@ryantxu is more of typescript catching a "bug", because I fixed the typing of ScopedVars so that properties can now be undefined. This code checked the scopedVars.__interval existed but then used scopedVars.__interval_ms , so just fixed the code so that it checks if scopedVars.__interval_ms exists instead (as that was what it wanted to access/use)